### DISCUSSION TITLE
How to send "pitch bend" from modulator???

### DISCUSSION THREAD

**Discussion by Ashankan** [RomanKubiak/ctrlr#685] [Q&A] [answered]
Hi.
Pretty straight forward question. I can send midi CC , but when I change the message type to "pitch bend" nothing happens.
But I can still read out the pitch bend in the midi monitor, but the midi does not go out.
Anyone have a quick fix for that?

**Comment by damiensellier** [comment]
I tried and it's working on my side : 

![Capture d’écran, le 2024-10-13 à 20 40 14](https://github.com/user-attachments/assets/e5e238e3-ad8b-4270-9f21-f0fb21f5e8cb)
![Capture d’écran, le 2024-10-13 à 20 39 38](https://github.com/user-attachments/assets/b125b572-1642-48d2-9715-105da9ec56e0)

It gives : 
![Capture d’écran, le 2024-10-13 à 20 39 50](https://github.com/user-attachments/assets/eb3f5d77-8b2d-4d48-a8ac-5ac404cd0e9b)

**Reply by Ashankan** [reply]
I thought it was something like that i was missing.

**Comment by damiensellier** [comment, marked as answer]
The MIDI Pitch Bend message is a 14-bit message with a total range of 0–16383. Since it's a bipolar control, it always starts at the center, 8192. Instead of displaying the starting point as 8192, it is displayed as 0, with positive and negative modulations of +/- 8191.

**Reply by Ashankan** [reply]
Thank you very much :)
